interactive_color.py

```diff
@@ -0,0 +1,751 @@
+import pygame
+
+""" The following section of the script initializes a few global varables that
+are helpful to reference at any point in the program."""
+
+pygame.init() # initialize pygame
+size = [1840, 920] # size of screen
+screen = pygame.display.set_mode(size)
+BLACK = (  0,   0,   0)
+WHITE = (255, 255, 255)
+BLUE =  (  0,   0, 255)
+GREEN = (  20, 255,   20)
+RED =   (255,   0,   0)
+color_matrix = [BLACK, BLUE, GREEN, RED]
+block_size = 40 # width/length of blocks in pixels
+jump = 0
+
```

Comment:
Try avoid putting a script outside and in between functions. All these 'scripts' should ideally go in your main() function that will be executed when this file runs.